tools/Customization/DevHome.Customization/ViewModels/Environment/DevDriveCardViewModel.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using CommunityToolkit.Mvvm.ComponentModel;
+using DevHome.Common.Models;
+using DevHome.Common.Services;
+using DevHome.SetupFlow.Utilities;
+
+using Dispatching = Microsoft.UI.Dispatching;
+
+namespace DevHome.Customization.ViewModels.Environments;
+
+/// <summary>
+/// View model for the card that represents a dev drive on the dev drive insights page.
+/// </summary>
+public partial class DevDriveCardViewModel : ObservableObject
+{
+    private readonly Dispatching.DispatcherQueue _dispatcher;
+
+    private readonly DevHome.Common.Services.IDevDriveManager _devDriveManager;
+
+    public string DevDriveLabel { get; set; }
+
+    public ulong DevDriveSize { get; set; }
+
+    public ulong DevDriveFreeSize { get; set; }
+
+    public ulong DevDriveUsedSize { get; set; }
+
+    public double DevDriveFillPercentage { get; set; }
+
+    public string DevDriveUnitOfMeasure { get; set; }
+
+    public string DevDriveSizeText { get; set; }
+
+    public string DevDriveUsedSizeText { get; set; }
+
+    public string DevDriveFreeSizeText { get; set; }
+
+    public DevDriveCardViewModel(IDevDrive devDrive, IDevDriveManager manager)
+    {
+        _dispatcher = Dispatching.DispatcherQueue.GetForCurrentThread();
+        _devDriveManager = manager;
+
+        DevDriveLabel = devDrive.DriveLabel;
+        var divider = (ulong)((devDrive.DriveUnitOfMeasure == ByteUnit.TB) ? 1000000000000 : 1000000000);

```

Comment:
Suggest separating the numbers out with underscores to make them easier to read.
https://stackoverflow.com/questions/43476056/what-do-the-underscores-mean-in-a-numeric-literal-in-c